packages/react/src/radiogroup/RadioGroup.stories.tsx

```diff
@@ -1,37 +1,139 @@
+import { useState, useEffect } from 'react';
 import type { Meta, StoryObj } from '@storybook/react';
 
-import { RadioGroup } from './RadioGroup';
+import { Button } from '../button/Button';
+import { RadioGroup, type RadioGroupProps } from './RadioGroup';
 import { Radio } from './Radio';
 
 const meta: Meta<typeof RadioGroup> = {
   title: 'RadioGroup',
   component: RadioGroup,
-  render: (args) => (
-    <RadioGroup {...args}>
-      <Radio value="ordinary">Ordinær pris</Radio>
-      <Radio value="bostart">OBOS Bostart</Radio>
-      <Radio value="deleie">OBOS Deleie</Radio>
-    </RadioGroup>
-  ),
+  argTypes: {
+    onChange: { action: 'changed' },
+  },
 };
 
 export default meta;
 
 type Story = StoryObj<typeof RadioGroup>;
 
-export const Example: Story = {
+const items = (
+  <>
+    <Radio value="ordinary">Ordinær pris</Radio>
+    <Radio value="bostart">OBOS Bostart</Radio>
+    <Radio value="deleie">OBOS Deleie</Radio>
+  </>
+);
+
+const Template = (args: RadioGroupProps) => {
+  return args.isRequired ? (
+    <form
+      className="flex flex-col items-start gap-4"
+      onSubmit={(e) => {
+        e.preventDefault();
+        alert('Lagret!');
+      }}
+    >
+      <RadioGroup {...args}>{items}</RadioGroup>
+      <Button type="submit">Send inn</Button>
+    </form>
+  ) : (
+    <RadioGroup {...args}>{items}</RadioGroup>
+  );
```

Comment:
Føltes litt random at `isRequired` styrer om det blir rendret som form med button eller ikke. Men det er kanskje ikke såå big deal siden det er snakk om intern Storybook-greie.

Reply:
Ja, er mest for å vise at du ikke får sendt inn når noe er required uten å gjøre et valg først. `isRequired` endrer ikke radiogruppen visuelt, så er mest for synliggjøring (og testing) i storybook.